Jellyfin.Plugin.JellyfinHelper/PluginPages/css/Recommendations.css

```
@@ -507,3 +507,274 @@
+    left: 0;
+    right: 0;
+    bottom: 0;
+    background: rgba(0, 0, 0, 0.6);
+    display: flex;
+    align-items: center;
+    justify-content: center;
+    z-index: 9999;
+    animation: popupFadeIn 0.15s ease;
+}
+
+@keyframes popupFadeIn {
```

Stylelint (17.10.0)

[error] Expected keyframe name "popupFadeIn" to be kebab-case (keyframes-name-pattern)

(keyframes-name-pattern)